wavefront_opentracing_python_sdk/tracer.py

```diff
@@ -0,0 +1,207 @@
+"""
+Wavefront Tracer.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+
+import time
+import uuid
+from opentracing import Tracer, Reference, ReferenceType, \
```

Comment:
Since we need so many things from `opentracing`, I'd better do `import opentracing` and then use `opentracing.Tracer`, etc.